### DISCUSSION TITLE
I could not figure out how to go from seg_logits to softmax

### DISCUSSION THREAD

**Discussion by ale2sia-g** [open-mmlab/mmsegmentation#3491] [Q&A]
Hi there. I am a beginner with MMSegmentation.
I am working on a 2D binary segmentation problem; in particular, 0 = background, 1 = tumor. To post-process the result from inference, I would like to work on the softmax as it was a gray-scale image with pixel values between 0 and 1, corresponding to the probability that the specific pixel belongs to the class 1. The real issue is that I just can't figure out how to obtain it from `results` or `seg_logits`.
After I do this:
```
result = inference_model(model, image_to_test)
seg_logits = result.seg_logits
```
I think I need some function to convert logits into probabilities. But I can't find the right function to use, as whatever function I use I get the error "'PixelData' object has no attribute..." etc.

Someone told me to give [this code](https://github.com/open-mmlab/mmsegmentation/blob/main/mmseg/models/segmentors/base.py#L188) a look but of course I'm not expert enough to figure it out on my own.

I am sure I am missing something. I would be grateful to get your help!

**Comment by changwsh12** [comment]
The logits can be directly used. Please refer to
https://github.com/open-mmlab/mmsegmentation/issues/3528#issuecomment-1974978592
